src/main/java/com/softserve/academy/spaced/repetition/domain/EntityName.java

```diff
@@ -0,0 +1,9 @@
+package com.softserve.academy.spaced.repetition.domain;
+
+import javax.persistence.*;
+import java.util.List;
+
+public enum EntityName{
+
+   DECK_COMMENT
```

Comment:
DECK_COMMENT("DECK_COMMENT") and use this value in @PreAuthorize (e.g. `@PreAuthorize("hasPermission('" +DECK_COMMENT.getValue()+ "','DELETE')")` )

Reply:
the same for Operations

Reply:
don't work in this way